shell/components/formatter/ClusterRepoBranch.vue

```diff
@@ -0,0 +1,29 @@
+<script>
+export default {
+  props: {
+    value: { // row
+      type:    Object,
+      default: () => {}
+    },
+    col: {
+      type:    Object,
+      default: () => {}
+    }
+  },
+
+  computed: {
+    clusterRepoBranch() {
+      return this.value?.isOciType ? this.t('generic.na') : this.value?.spec?.gitBranch ? this.value.spec.gitBranch : undefined;
```

Comment:
We might want to pause on this. 

When the list is updated to use server-side pagination the sort and search will be broken (given both will still be on value.spec.gitBranch).

The alternative is to not use server-side pagination for this list, given it's small

Reply:
Even without this change we can have missing gitBranch in the `spec` object. In those cases we currently show `–` for all types of cluster repos, but here we show `n/a` for just the oci type, is there a way to test the server-side pagination now?

Reply:
Server side sort and filtering can only be done on values that natively exist in the resource server side.

Here we would be showing `n/a` but still sorting and filtering on `spec.gitBranch`'s undefined value. Sorting might seem ok UX wise but wouldn't strictly be true (aaa1, na1, na2, x1, n/a, n/a, n/a, n/a). The user wouldn't wouldn't get any results if they filtered on n/a.

Maybe it's just not worth converting this list. Maybe one for UX
1) Support both server-side paginated lists and non-server-side pagination lists in the same session
2) Support only server-side paginated lists but remove functionality like above